torch_xla/experimental/mark_pattern_utils.py

```diff
@@ -0,0 +1,86 @@
+import uuid
+from typing import Dict, Union
+
+import torch
+from torch_xla.experimental import xla_marker
+
+
+def _xla_mark_tensor(*args, **kwargs):
+  return torch.ops.xla_pattern_marking.mark_tensor(*args, **kwargs)
+
+
+class StableHLOCompositeBuilder:
+
+  def __init__(self,
+               name: str,
+               attrs: Dict[str, Union[int, float, str]] = None):
+
+    self.attrs = attrs
+    self.name = str(name)
```

Comment:
why copy the name? str is immutable anyways

Reply:
refactored the impl